### PR TITLE
157 update board interface

### PR DESCRIPTION
**Pull request by iandareid**
The interface changes have been made. They assume that the sensors always have new data when queried. This hasn't caused strange behavior, and I believe that is how it was done before changing the interface. Just something that seems odd.

### CODE REVIEW

**Comment by iandareid** [comment]
@JMoore5353 @bsutherland333 

I am curious as to your thoughts with the having the SENSOR_has_new_data() methods always returning true.

**Comment by iandareid** [comment]
There was a bug with the return true, so I adjusted it to match the imu_new_data which uses gazebo time. This can be easily switched to generic sim time when we finally disconnect gazebo.

File: rosflight_sim/src/sil_board.cpp [hunk 1]
```diff
@@ -70,6 +70,16 @@ void SILBoard::gazebo_setup(gazebo::physics::LinkPtr link, gazebo::physics::Worl
   auto remote_host = node_->get_parameter_or<std::string>("ROS_host", "localhost");
   int remote_port = node_->get_parameter_or<int>("ROS_port", 14520);
 
+  // IMU timer
```
Comment (bsutherland333) [inline rosflight_sim/src/sil_board.cpp:73]: Are you intending to leave these comments here? That's fine if sure, just wanted to make sure you didn't forget them

**Review by bsutherland333 — APPROVED** [review]
Looks good, I just had one comment

File: rosflight_sim/src/sil_board.cpp (hunk 1 above)
Comment (iandareid) [reply rosflight_sim/src/sil_board.cpp:73]: Hahaha nope thats where I was going to put them.

**Pull request merged**